README.rst

```diff
@@ -42,7 +42,7 @@ You can get help with this code on our `mailing lists`_ or in real-time conversa
 .. _mailing lists: https://open.edx.org/getting-help
 .. _Slack: https://open.edx.org/getting-help
 
-.. |build-status| image:: https://travis-ci.org/edx/edx-ora2.png?branch=master
+.. |build-status| image:: https://travis-ci.com/edx/edx-ora2.svg?branch=master
    :target: https://travis-ci.org/edx/edx-ora2
```

Comment:
can you also update the `target` to point to travis-ci.com instead

Reply:
Okay